project/context.go

```diff
@@ -96,9 +99,14 @@ func (c *Context) lookupProjectName() (string, error) {
 		return envProject, nil
 	}
 
-	f, err := filepath.Abs(c.ComposeFile)
+	file := ""
```

Comment:
I would use `file := "."` to show that it takes the current directory by default — it's a nit, really :stuck_out_tongue_closed_eyes: .